pkgs/top-level/php-packages.nix

```diff
@@ -1,10 +1,12 @@
 { pkgs, php }:
 
-let self = with self; {
-  buildPecl = import ../build-support/build-pecl.nix {
-    inherit php;
-    inherit (pkgs) stdenv autoreconfHook fetchurl;
-  };
+let
+  self = with self; {
+    buildPecl = import ../build-support/build-pecl.nix {
+      inherit php;
+      inherit (pkgs) stdenv autoreconfHook fetchurl;
+    };
+  php7 = pkgs.lib.versionAtLeast php.version "7.0";
```

Comment:
@shlevy I copied that check from the `php` package. TBH I have still very little knowledge of the nix language. I just need to check whether the given `php` package is version 7.0+
Can you help me getting this right?